Client/Frontend/Brave Rewards/OptIn/BraveTalkRewardsOptInView.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveUI
+import Shared
+import BraveShared
+
+extension BraveTalkRewardsOptInViewController {
+    class View: UIView {
+        
+        let enableRewardsButton = ActionButton().then {
+            $0.layer.borderWidth = 0
+            $0.titleLabel?.font = .systemFont(ofSize: 16.0, weight: .semibold)
```

Comment:
Use preferred fonts/scaled fonts and set `adjustsFontForContentSizeCategory` to true